code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -207,14 +207,18 @@
 		R.hat = null
 	R.update_headlamp()

```

Comment:
you need to inherit this line, the three lines above it, and the `R.notransform = TRUE` seven lines above it.

Reply:
no, copypasting it is not acceptable

Reply:
The only downside to this is now it gives off that smoke puff which I'd rather not have during the animations, I'll swap it to the ResetModule instead